src/transforms/groupby.js

```diff
@@ -35,14 +35,24 @@ exports.attributes = {
         ].join(' ')
     },
     style: {
```

Comment:
right, at the moment `_arrayAttrRegexps` doesn't do anything inside traces, it's just a question of if we want to put it in so we don't forget about this when we *do* get to implementing it. Maybe better not to include it, in case we implement it a different way. I think `_isLinkedToArray` matters to [plot_schema](https://github.com/plotly/plotly.js/blob/master/src/plot_api/plot_schema.js) already though, doesn't it?